Rubberduck.Refactorings/EncapsulateField/EncapsulateFieldRefactoring.cs

```diff
@@ -91,26 +90,26 @@ protected override EncapsulateFieldModel InitializeModel(Declaration target)
 
         protected override void RefactorImpl(EncapsulateFieldModel model)
         {
-            var refactorRewriteSession = new EncapsulateFieldRewriteSession(_rewritingManager.CheckOutCodePaneSession()) as IEncapsulateFieldRewriteSession;
+            var refactorRewriteSession = _rewritingManager.CheckOutCodePaneSession();
 
-            refactorRewriteSession = RefactorRewrite(model, refactorRewriteSession);
+            RefactorRewrite(model, refactorRewriteSession as IRewriteSession);
```

Comment:
The cast seems to be redundant; the return type of `CheckOutCodePaneSession` extends `IRewrtiteSession`.
Or is there another overload taking an `IExecutableRewriteSession`?